docs/reference_guide.md

```diff
@@ -1268,7 +1268,18 @@ hi
 ^C
 ```
 
-bpftrace also supports USDT semaphores. You may activate semaphores by passing in `-p $PID` or
+bpftrace also supports USDT semaphores. If both your environment and bpftrace
```

Comment:
Is this a (new) kernel feature or has it been in there for ages? Might be good to list the required version if this requires a modern kernel